eng/tools/specs-model/package.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "@azure-tools/specs-model",
+  "private": true,
+  "type": "module",
+  "main": "dist/src/index.js",
+  "engines": {
+    "node": ">= 18.0.0"
+  },
+  "scripts": {
+    "build": "tsc",
+    "postinstall": "npm run build",
+    "test": "vitest",
+    "test:ci": "vitest run --coverage --reporter=verbose",
+    "lint": "eslint . -c eslint.config.js --report-unused-disable-directives --max-warnings 0",
+    "lint:fix": "eslint . -c eslint.config.js --fix",
+    "prettier": "prettier . --check",
+    "prettier:debug": "prettier . --check ---log-level debug",
+    "prettier:write": "prettier . --write"
+  },
+  "bin": {
+    "get-specs-model": "cmd/get-specs-model.js"
+  },
+  "dependencies": {
+    "lodash": "^4.17.21"
+  },
+  "devDependencies": {
+    "@eslint/js": "^9.7.0",
+    "@tsconfig/strictest": "^2.0.5",
+    "@types/eslint__js": "^8.42.3",
+    "@types/lodash": "^4.17.6",
+    "@types/node": "^20.14.10",
+    "@vitest/coverage-v8": "^2.0.2",
+    "eslint": "^9.7.0",
```

```suggestion
    "eslint": "^8.57.0",
```

I think we should stay on `eslint` v8 until `typescript-eslint` supports v9.  This also aligns with `microsoft/typespec`.